nixos/modules/services/misc/ersatztv.nix

```diff
@@ -108,7 +112,15 @@ in
           ETV_CONFIG_FOLDER = "/var/lib/ersatztv/config";
           ETV_TRANSCODE_FOLDER = "/var/lib/ersatztv/transcode";
         }
-        // cfg.environment;
+        // (lib.mapAttrs (
+          _: s:
+          if (lib.isAttrs s || lib.isList s) then
```

Comment:
Does ErsatzTV avtually expect any list/attrsets in its environment variables? The option type doesn't allow them anyway so this is dead code.